File: simulator/project-python/common/order/order.py
import json


class Order:
    user_id: str
    order_id: str
    amount: float

    def __init__(self, user_id, order_id, amount):
        self.user_id = user_id
        self.order_id = order_id
        self.amount = amount

    def serializer(self):
        return json.dumps(
            {"user_id": self.user_id, "order_id": self.order_id, "amount": self.amount}
        )

File: simulator/project_java/service-user/src/main/java/service/user/Order.java
package service.user;

import java.math.BigDecimal;

public class Order {
  private final String userId, orderId;
  private final BigDecimal amount;

  public Order(String userId, String orderId, BigDecimal amount) {
    this.userId = userId;
    this.orderId = orderId;
    this.amount = amount;
  }

  public BigDecimal getAmount() {
    return this.amount;
  }

  public String getUserId() {
    return this.userId;
  }

  public String getOrderId() {
    return this.orderId;
  }

  public String getEmail() {
    return "";
  }
}